app.json

```diff
@@ -113,7 +113,7 @@
   },
   "addons": [
     "attachment-scanner",
-    "heroku-postgresql",
+    {"plan": "heroku-postgresql", "options":  {"version": "10"}},
```

Comment:
👍

Reply:
This only works for the review apps I think - not all of our stuff everywhere. Sadly.